docs/quickstart.md

```diff
@@ -0,0 +1,293 @@
+
+This tutorial will introduce the main `nemos` functionalities. This is intended for users that are 
+already familiar with the GLM framework but want to learn how to interact with the `nemos` API. 
+If you have used [scikit-learn](https://scikit-learn.org/stable/) before, we are compatible the [estimator API](https://scikit-learn.org/stable/modules/generated/sklearn.base.BaseEstimator.html), so the quickstart should look 
+familiar.
+
+In the following sessions, you will learn:
+
+1. How to define and fit a GLM model.
+2. Which are the GLM parameters and hyperparameters and how to set it.
+3. How `nemos` interacts with `pynapple` for pre-processing
+4. How `nemos` interacts with `scikit-learn` for pipelines and cross-validation. 
```

Comment:
```suggestion
1. [How to define and fit a GLM model.](#basic-model-fitting)
2. [How to access and set the GLM parameters and optimization hyperparameters.](#model-arguments)
3. [How `nemos` interacts with `pynapple` for pre-processing.](#interactions-with-pynapple)
4. [How `nemos` interacts with `scikit-learn` for pipelines and cross-validation.](#interactions-with-scikit-learn)
```
Adds links, slight rewording of point number 2 (to preserve structure)

Note that the links being correct depends on the heading renaming I do below